<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NfsConfig</class>
 <widget class="QDialog" name="NfsConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NfsConfig</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../images.qrc">:/images/nfs.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Directory NFS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Directory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditDir">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonDir">
          <property name="text">
           <string>Open directory</string>
          </property>
          <property name="icon">
           <iconset resource="../images.qrc">
            <normaloff>:/images/directory.png</normaloff>:/images/directory.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Host access&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Access</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditAccess"/>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Read/Write</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxRW">
          <item>
           <property name="text">
            <string>ro</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>rw</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Syncronized</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditSync"/>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Root</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxSquash">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>root_squash</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>no_root_squash</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>all_squash</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>no_all_squash</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>AnonUID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditAnonUID"/>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>AnonGID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditAnonGID"/>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Guest</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditGuest"/>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Structure</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxSub">
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>subtree_check</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>no_subtree_check</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/images/document-save.png</normaloff>:/images/document-save.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditDir</tabstop>
  <tabstop>pushButtonDir</tabstop>
  <tabstop>lineEditAccess</tabstop>
  <tabstop>comboBoxRW</tabstop>
  <tabstop>lineEditSync</tabstop>
  <tabstop>comboBoxSquash</tabstop>
  <tabstop>lineEditAnonUID</tabstop>
  <tabstop>lineEditAnonGID</tabstop>
  <tabstop>lineEditGuest</tabstop>
  <tabstop>pushButtonSave</tabstop>
  <tabstop>pushButtonClose</tabstop>
 </tabstops>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
